packages/client/src/common/utils/theem.ts

```diff
@@ -0,0 +1,17 @@
+import { createTheme } from "@mui/material";
+
+export const customTheme = createTheme({
+  palette: {
+    primary: {
+      main: "#2e7d32 ",
+    },
+    secondary: {
+      main: "#DEC19B",
+    },
+    tonalOffset: 0.2,
+  },
+  typography: {
+    fontFamily: "sans-serif",
+    fontSize: 12,
+  },
```

Comment:
add direction: i18n.dir(), to theme for rtl support

Reply:
I can't use the hook in this file, it needs to be a component as react hooks said.
I used it in the app.tsx file.
@mohannadjamal